actions.go

```diff
@@ -2,54 +2,44 @@ package main
 
 import (
 	"fmt"
+	"log"
 	"net/http"
-	"os"
 )
 
 // PerformScan runs a scan of a given host using a set of scripts.
 // At this moment, we assume that there will be only one script here (idrac.py),
 // and that only MAC addresses will be created/updated/deleted in Ralph.
-func PerformScan(addrStr string, scripts []string, dryRun bool, cfgDir string) {
+func PerformScan(addrStr string, scripts []string, dryRun bool, cfg *Config, cfgDir string) {
 	if dryRun {
-		fmt.Println("Running in dry-run mode, no changes will be saved in Ralph.")
+		// TODO(xor-xor): Wire up logger here.
```

Comment:
Not everything at once - there's a separate task for logger etc. (see RAS-652 in our internal backlog).